dlt/destinations/impl/filesystem/sql_client.py

```diff
@@ -168,12 +170,17 @@ def _escape_column_name(col_name: str) -> str:
             first_file_type = self.get_file_format(table_schema)
 
             # build files string
-            supports_wildcard_notation = not self.is_abfss
+            supports_wildcard_notation = not self.is_abfss and is_folder
 
             resolved_files_string = f"'{table_location}/**/*.{first_file_type}'"
+
             if not supports_wildcard_notation:
+                # list_table_files returns a list of absolute paths but without scheme
                 files = self.remote_client.list_table_files(table_name)
-                resolved_files_string = ",".join(map(lambda f: f"'{protocol}://{f}'", files))
+                if protocol == "file":
```

Comment:
was there something wrong with leaving the protocol for "file"? maybe on windows? I'm asking because I recently spent a lot of time fixing this place. but you seem to be right here!

btw. where we really lack docstiring is FsClient. does `list_table_files` return absolute path? with scheme or not?